.github/workflows/tag-release.yml

```diff
@@ -1,36 +1,68 @@
-name: Create Releases on Tags
+name: Auto tag-release-publish
 
 on:
   push:
-    tags:
-      - 'v*'
+    branches:
+      - master
```

Comment:
Is this going to tag a new release on each merge to master?

Reply:
nope! it will only tag a new release when the version in `package.json` is updated